src/allowed_args.cpp

```diff
@@ -561,6 +561,8 @@ static void addNodeRelayOptions(AllowedArgs &allowedArgs)
         .addArg("datacarriersize=<n>", requiredInt,
             strprintf(_("Maximum size of data in data carrier transactions we relay and mine (default: %u)"),
                     MAX_OP_RETURN_RELAY))
+        .addArg("dust-threshold=<amt>", requiredAmount,
+            strprintf(_("Dust Threshold (in satoshis) used to defined dust. (default: %s)"), DEFAULT_DUST_THRESHOLD))
```

Comment:
please don't use the word in the definition.  How about:

"Dust Threshold (in satoshis) defines the minimum quantity an output may contain for the transaction to be considered 'standard' (relayable)"

Reply:
ok, my bad...will change that.